assets/src/setup/pages/choose-reader-theme/index.js

```diff
@@ -62,17 +84,38 @@ export function ChooseReaderTheme() {
 				}
 			</p>
 			<form>
-				<ul className="choose-reader-theme__grid">
-					{
-						themes && themes.map( ( theme ) => (
+				{ 0 < availableThemes.length && (
+					<ul className="choose-reader-theme__grid">
+						{ availableThemes.map( ( theme ) => (
 							<ThemeCard
 								key={ `theme-card-${ theme.slug }` }
 								screenshotUrl={ theme.screenshot_url }
 								{ ...theme }
-							/> ),
-						)
-					}
-				</ul>
+							/>
+						) ) }
+					</ul>
+				) }
+
+				{ 0 < uninstallableThemes.length && (
+					<div className="choose-reader-theme__uninstallable">
+						<h3>
+							{ __( 'Uninstallable themes', 'amp' ) }
```

Comment:
Could you please rename `uninstallable` to `unavailable` (or `nonInstallable`) across the codebase (e.g. in the in this file and in `ThemeCard`)? I realized that this is misleading: like it's telling you the list of themes which you can uninstall, but that's not what we intend! 😄